cs_practical_examples/addtoany/share_after.html

```diff
@@ -0,0 +1 @@
+../../add_to_any/share_after.html
```

Comment:
@giniyat202 is this a symlink?

Reply:
Yes. to make development easier I grouped all examples that were scattered around in a single directory `cs_practical_examples`, with each example being placed under a subdirectory named after the provided. I choose to make symlinks instead of moving the examples to avoid:

1. breaking the links article once this branch is merged
1. breaking links in other articles that might be using the same examples

Reply:
@giniyat202 I was exactly questioning if github would have symlink correctly served, glad you checked it first.
I'd go for 3 (copy to new and delete the old one), but we have to make sure the old locations aren't referenced anywhere else in our guides.
So I'd say we can proceed this way:
1. copy snippets to new location
2. make a list of all the old locations to delete (maybe it might be in the README.md so we can also say that those snippets are deprecated and `cs_practical_examples` should be updated instead)
3. have Silvio update the guides with the new locations
4. have Silvio check if there are any other references to the list in step 2
5. delete the old locations

Probably this will take some time and will be done in different tasks (current task for steps 1-2, task for step 3, task for step 4, task for step 5) but IMHO it's the safest way to go that still allows us to keep things organized.
Wdyt?

Reply:
@matita did 1 and 2 (had to do a rebase, sorry for that, please do a `git pull -f` when checking out the branch). I also updated review instruction in the comment below and added a link to view (and review) the edits on github. so, can you please take another look?